### PR TITLE
Add OpenSea icon

### PR DESCRIPTION
**Pull request by hammy2899**
opensea

**Issue: https://github.com/simple-icons/simple-icons/issues/6773**
**Alexa rank: 205**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Opened due to https://github.com/simple-icons/simple-icons/pull/6148 being closed for some time.

HEX value is derived from the logo page on the OpenSea website. [https://docs.opensea.io/docs/logos](https://docs.opensea.io/docs/logos).
I converted the `Transparent Logomark` icon from that page to SI format.

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
Hi @hammy2899, thanks for the contribution.

I see some differences, maybe up the precision to 4 will help

image

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7016,6 +7016,11 @@
             "hex": "0097C2",
             "source": "https://opennebula.io/docs/"
         },
+        {
+            "title": "OpenSea",
+            "hex": "2081E2",
+            "source": "https://docs.opensea.io/docs/logos"
```
Comment (dirien) [inline _data/simple-icons.json:7022]: can you change the source to `https://docs.opensea.io` only please.

Comment (hammy2899) [reply _data/simple-icons.json]: I've changed the url in latest commit.

**Comment by hammy2899** [comment]
@dirien I've reoptimized the svg yet still have differences even with precision 8.

Precision 8:
image

**Comment by dirien** [comment]
@service-paradis, what do you think? Should we rollback to default precission an merge the PR?

**Comment by jorgeamadosoria** [comment]
I've used the transparent logomark and after removing the fill, resizing to 24x24 and optimized it with svgo at precision 3 I got
`<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>OpenSea</title><path d="M12 0C5.374 0 0 5.374 0 12s5.374 12 12 12 12-5.374 12-12S18.629 0 12 0ZM5.92 12.403l.051-.081 3.123-4.884a.107.107 0 0 1 .187.014c.52 1.169.972 2.623.76 3.528-.088.372-.335.876-.614 1.342a2.405 2.405 0 0 1-.117.199.106.106 0 0 1-.09.045H6.013a.106.106 0 0 1-.091-.163zm13.914 1.68a.109.109 0 0 1-.065.101c-.243.103-1.07.485-1.414.962-.878 1.222-1.548 2.97-3.048 2.97H9.053a4.019 4.019 0 0 1-4.013-4.028v-.072c0-.058.048-.106.108-.106h3.485c.07 0 .12.063.115.132-.026.226.017.459.125.67.206.42.636.682 1.099.682h1.726v-1.347H9.99a.11.11 0 0 1-.089-.173l.063-.09c.16-.231.391-.586.621-.992.156-.274.308-.566.43-.86.024-.052.043-.107.065-.16.033-.094.067-.182.091-.269a4.57 4.57 0 0 0 .065-.223c.057-.25.081-.514.081-.787 0-.108-.004-.221-.014-.327-.005-.117-.02-.235-.034-.352a3.415 3.415 0 0 0-.048-.312 6.494 6.494 0 0 0-.098-.468l-.014-.06c-.03-.108-.056-.21-.09-.317a11.824 11.824 0 0 0-.328-.972 5.212 5.212 0 0 0-.142-.355c-.072-.178-.146-.339-.213-.49a3.564 3.564 0 0 1-.094-.197 4.658 4.658 0 0 0-.103-.213c-.024-.053-.053-.104-.072-.152l-.211-.388c-.029-.053.019-.118.077-.101l1.32.357h.01l.173.05.192.054.07.019v-.783c0-.379.302-.686.679-.686a.66.66 0 0 1 .477.202.69.69 0 0 1 .2.484V6.65l.141.039c.01.005.022.01.031.017.034.024.084.062.147.11.05.038.103.086.165.137a10.351 10.351 0 0 1 .574.504c.214.199.454.432.684.691.065.074.127.146.192.226.062.079.132.156.19.232.079.104.16.212.235.324.033.053.074.108.105.161.096.142.178.288.257.435.034.067.067.141.096.213.089.197.159.396.202.598a.65.65 0 0 1 .029.132v.01c.014.057.019.12.024.184a2.057 2.057 0 0 1-.106.874c-.031.084-.06.17-.098.254-.075.17-.161.343-.264.502-.034.06-.075.122-.113.182-.043.063-.089.123-.127.18a3.89 3.89 0 0 1-.173.221c-.053.072-.106.144-.166.209-.081.098-.16.19-.245.278-.048.058-.1.118-.156.17-.052.06-.108.113-.156.161-.084.084-.15.147-.208.202l-.137.122a.102.102 0 0 1-.072.03h-1.051v1.346h1.322c.295 0 .576-.104.804-.298.077-.067.415-.36.816-.802a.094.094 0 0 1 .05-.03l3.65-1.057a.108.108 0 0 1 .138.103z"/></svg>`

which is a perfect match with the source logomark. Please check it on your side @dirien and @hammy2899. As far as I can see, there is no need for precision 4 or above.

**Comment by sachinraja** [comment]
I checked @jorgeamadosoria version and it matches perfectly.

File: _data/simple-icons.json (hunk 1 above)
Comment (sachinraja) [reply _data/simple-icons.json]: I think it makes sense to link to https://docs.opensea.io/docs/logos. That seems like a fairly permanent link.

**Comment by hammy2899** [comment]
I also verified @jorgeamadosoria's and it seems good so I've updated to PR.

**Comment by dirien** [comment]
LGTM

**Pull request merged**